src/index.js

```diff
@@ -176,10 +176,14 @@ var cronofy = function(config){
           resolve(result['entity']);
         }
       }, function(err){
+        var error = {
```

Comment:
I would prefer a real Error object here.
Is err.entity a real error? if so just add the status code to that and give that back